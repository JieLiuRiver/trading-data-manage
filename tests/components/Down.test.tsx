import React from 'react';
import { render } from '@testing-library/react';
import DownIcon from '@/components/SvgIcons/Down';

describe('DownIcon', () => {
  it('should render correctly with default props', () => {
    const { container } = render(<DownIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render correctly with size and color props', () => {
    const { container } = render(<DownIcon size={48} color="red" />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
