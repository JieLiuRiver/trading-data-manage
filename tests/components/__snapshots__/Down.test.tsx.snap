// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownIcon should render correctly with default props 1`] = `
<svg
  class="icon"
  height="24"
  p-id="9719"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M866.742857 762.148571l-20.48-170.788571a6.948571 6.948571 0 0 0-11.702857-4.022857l-50.834286 51.2-232.594285-232.594286a14.262857 14.262857 0 0 0-19.382858 0l-87.04 87.04L207.36 256a6.948571 6.948571 0 0 0-9.508571 0L159.085714 294.765714a6.948571 6.948571 0 0 0 0 9.508572l275.748572 276.114285a13.531429 13.531429 0 0 0 19.382857 0l87.405714-87.405714 193.828572 193.828572-50.834286 50.834285a6.948571 6.948571 0 0 0 4.022857 11.702857l170.422857 20.48a6.582857 6.582857 0 0 0 7.68-7.68z"
    p-id="9720"
  />
</svg>
`;

exports[`DownIcon should render correctly with size and color props 1`] = `
<svg
  class="icon"
  fill="red"
  height="48"
  p-id="9719"
  version="1.1"
  viewBox="0 0 1024 1024"
  width="48"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M866.742857 762.148571l-20.48-170.788571a6.948571 6.948571 0 0 0-11.702857-4.022857l-50.834286 51.2-232.594285-232.594286a14.262857 14.262857 0 0 0-19.382858 0l-87.04 87.04L207.36 256a6.948571 6.948571 0 0 0-9.508571 0L159.085714 294.765714a6.948571 6.948571 0 0 0 0 9.508572l275.748572 276.114285a13.531429 13.531429 0 0 0 19.382857 0l87.405714-87.405714 193.828572 193.828572-50.834286 50.834285a6.948571 6.948571 0 0 0 4.022857 11.702857l170.422857 20.48a6.582857 6.582857 0 0 0 7.68-7.68z"
    p-id="9720"
  />
</svg>
`;
